docker-compose.dev.yml

```diff
@@ -10,7 +10,9 @@ services:
   database:
     restart: "no"
   migrations:
-    restart: "no"  
+    restart: "no"
+    volumes:
+      - ./config/dev-migrations:/data/migrations/dev
```

Comment:
We usually use `./config/migrations/development` as folder and mount it in `/data/migrations/development`